pattern-library/sass/patterns/_forms.scss

```diff
@@ -260,12 +260,13 @@ $input-width: (
 
     // Input fields immediately followed by a button (ie, search boxes) should eliminate the spacing
     // between them and adopt matching styling/sizing to emphasize that they are connected.
-    &[type=search] + button {
-        border-top-left-radius: 0;
-        border-bottom-left-radius: 0;
+    + button {
```

Comment:
This still needs to be `&[type=search] + button {`. Otherwise it applies to all buttons in `.input-text`, not just buttons that immediately follow search boxes.